SPECS-EXTENDED/volume_key/volume_key.spec

```diff
@@ -157,6 +166,10 @@ exit 1; \
 %endif
 
 %changelog
+* Wed Nov 06 2024 Jyoti Kanase <v-jykanase@microsoft.com> - 0.3.12-9
+- added patch & miner fixes
```

Comment:
It is better if we can add the details of the patch and its need in a line or two.